src/common/options/global.yaml.in

```diff
@@ -3592,7 +3592,7 @@ options:
 - name: osd_pg_delete_cost
   type: size
   level: advanced
-  default: 1_M
+  default: 15_M
```

Comment:
This will have implications if users revert to wpq for some reason. I'd suggest adding a method similar to ` maybe_override_options_for_qos()`, for e.g. `maybe_override_cost_for_qos()` that overrides the set of costs that will work with mclock.

As discussed, before adding the above, can you clarify if the result charts you have included is with the current best effort allocations? The current allocations for high_client_ops are:
 res: 25% | wgt: 2 | limit: max |

Did you try running the test with defaut cost (1_M) and the new cost with the modified allocations for best effort client? The new allocations are:
res: 25% | wgt: 1 | lim: 30% |

You can share the data on this offline and we can proceed from there.

Reply:
I double checked the tests we had run and the second chart I have included shows the PG deletion rate for WPQ and the high_client_ops profile with the modified allocations you have mentioned. 
I am attaching another graph that shows the deletion trends for default high_client_ops profile with default cost and the modified allocations with default cost, this will show why we needed the cost modification as well.

Reply:
@amathuria Thanks for the clarification. I think you can introduce the new method to override the cost. We can utilize this method to override costs for other ops based on the need.